### PR TITLE
[NO-ISSUE] Upgrade org.json jar

### PR DESCRIPTION
**Pull request by pjfanning**
### What is this PR for?

See ZEPPELIN-5996

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Task

### What is the Jira issue?
* [ZEPPELIN-5996](https://issues.apache.org/jira/browse/ZEPPELIN-5996)

### How should this be tested?
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the license files need to update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

**Comment by jongyoul** [comment]
Don't we have to update the LICENSE file?

File: flink/flink-scala-2.12/pom.xml [hunk 1]
```diff
@@ -254,6 +254,17 @@
       <groupId>com.mashape.unirest</groupId>
       <artifactId>unirest-java</artifactId>
```
Comment (Reamer) [inline flink/flink-scala-2.12/pom.xml:255]: I think it makes more sense to raise this dependency and not play around with the transitive dependency.
From version 3.0 this library uses  `com.google.code.gson » gson` and no longer `org.json » json`.
We also use GSON in many other places in Zeppelin.

**Review by Reamer — COMMENTED** [review]
I have also checked the other dependencies. The JSON dependency has been dropped and has mostly been replaced by GSON.

File: flink/flink-scala-2.12/pom.xml (hunk 1 above)
Comment (pjfanning) [reply flink/flink-scala-2.12/pom.xml:255]: I am not a Zeppelin committer. I never use Zeppellin. I'm just trying to help out but have very limited time to be experimenting.

**Comment by pjfanning** [comment]
> Don't we have to update the LICENSE file?

I've just updated [zeppelin-distribution/src/bin_license/LICENSE](https://github.com/apache/zeppelin/pull/4722/files#diff-73d534459c803c1cd56450d7b0386aeff4b1f0ecffb3a80a0bc00c0d053a57c6)

**Comment by pjfanning** [comment]
@Reamer @jongyoul could this PR be merged? Zeppelin is breaking ASF rules by using a Category X dependency.

**Review by jongyoul — APPROVED** [review]

**Pull request merged**